ecosystem/sep-0010.md

```diff
@@ -309,9 +309,11 @@ This setup allows the **Server** to verify that the challenge returned by the **
 
 ### Memos
 
-Stellar transaction memos are used for a variety of purposes in the ecosystem, but in the context of this standard memos attached to challenge transactions are typically used to distinguish users who share a single Stellar account. This is especially relevant for applications who custody user funds in shared Stellar accounts.
+Stellar transaction memos are used for a variety of purposes in the ecosystem, but in the context of this standard memos attached to challenge transactions distinguish sessions that should be entirely separate, scoped, and detached from each other. Two users who are authenticated with the same Stellar account but different memos should have the same level of separation as two users who are authenticated with different Stellar accounts. Typically the sessions are unique users who share a single Stellar account, sometimes called an omnibus or pooled account.
 
-The `memo_type` and `memo` parameters supported in `GET <WEB_AUTH_ENDPOINT>` API calls can be used to communicate to the **Server** that the client intends to authenticate a Stellar account with an additional claim that the account is shared and that the user should be identified using the `account` and `memo` parameters, instead of only using `account`.
+The `memo` parameter supported in `GET <WEB_AUTH_ENDPOINT>` API calls can be used to communicate to the **Server** that the client intends to authenticate a Stellar account with an additional claim that the account is shared and that the user should be identified using the `account` and `memo` parameters, instead of only using `account`. The value of the `memo` passed will ultimately be added to the decoded JWT's `sub` field, separated from the account address by a colon (`:`).
+
+Note that conceptually, a Stellar account and memo (of type `id`) is equivalent to a muxed account (`M...`) defined by [CAP-0027](../core/cap-0027.md). If a SEP-10 implementation supports the use of memos to identify users of shared accounts, it is highly recommended to also support the muxed account address format. **Muxed accounts will become the primary (and eventually exclusive) method for identifying a user of a shared, omnibus, or pooled account.**
```

Comment:
💡 I think we should reserve bolding for entities in SEP-10. This sentence is important, but not necessarily more important than other text in the document. 
```suggestion
Note that conceptually, a Stellar account and memo (of type `id`) is equivalent to a muxed account (`M...`) defined by [CAP-0027](../core/cap-0027.md). If a SEP-10 implementation supports the use of memos to identify users of shared accounts, it is highly recommended to also support the muxed account address format. Muxed accounts will become the primary (and eventually exclusive) method for identifying a user of a shared, omnibus, or pooled account.
```